File: Desenvolvimento/class/pedidoProduto.php
<?php

/**
 * Created by PhpStorm.
 * User: André
 * Date: 27/04/2017
 * Time: 21:32
 */
class pedidoProduto{
    private $produto;
    private $pedidoCorte;
    private $largura;
    private $altura;
    private $qtd_chapas;

    public function setProduto($produto){
        $this->produto = $produto;
    }
    public function getProduto(){
        return $this->produto;
    }
    public function setPedidoCorte($pedidoCorte){
        $this->pedidoCorte = $pedidoCorte;
    }
    public function getPedidoCorte(){
        return $this->pedidoCorte;
    }
    public function setLargura($largura){
        $this->largura = $largura;
    }
    public function getLargura(){
        return $this->largura;
    }
    public function setAltura($altura){
        $this->altura = $altura;
    }
    public function getAltura(){
        return $this->altura;
    }
    public function setQtd_chapas($qtd_chapas){
        $this->qtd_chapas = $qtd_chapas;
    }
    public function getQtd_chapas(){
        return $this->qtd_chapas;
    }
}

File: Sistema de Corte de Rochas - Caso de Uso/src/javafx/domain/PedidoProduto.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package javafx.domain;

/**
 *
 * @author André
 */
public class PedidoProduto {
    private PedidoDeCorte pedidoDeCorte;
    private Produto produto;
    private float largura;
    private float altura; 
    private String qtd;

    public PedidoDeCorte getPedidoDeCorte() {
        return pedidoDeCorte;
    }

    public void setPedidoDeCorte(PedidoDeCorte pedidoDeCorte) {
        this.pedidoDeCorte = pedidoDeCorte;
    }

    public Produto getProduto() {
        return produto;
    }

    public void setProduto(Produto produto) {
        this.produto = produto;
    }

    public float getLargura() {
        return largura;
    }

    public void setLargura(float largura) {
        this.largura = largura;
    }

    public float getAltura() {
        return altura;
    }

    public void setAltura(float altura) {
        this.altura = altura;
    }

    public String getQtd() {
        return qtd;
    }

    public void setQtd(String qtd) {
        this.qtd = qtd;
    }
    
    
}
